docker-compose-services.yaml

```
@@ -0,0 +1,42 @@
+  #   volumes:
+  #     - influxdb_data:/var/lib/influxdb2
+  #   environment:
+  #     DOCKER_INFLUXDB_INIT_MODE: setup
+  #     DOCKER_INFLUXDB_INIT_USERNAME: user
+  #     DOCKER_INFLUXDB_INIT_PASSWORD: password
+  #     DOCKER_INFLUXDB_INIT_ORG: org
+  #     DOCKER_INFLUXDB_INIT_BUCKET: bucket
+  #     DOCKER_INFLUXDB_INIT_ADMIN_TOKEN: my-token
+
+volumes:
+  influxdb_data:
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)